pyrog-schema.yml

```diff
@@ -0,0 +1,166 @@
+openapi: 3.0.2
```

Comment:
there are only success return codes, will it auto-generate error codes when we'll add actual implementations in the views ?

Reply:
It will not. Schema generation does introspect actual implementation.
What we can do is add a default error response for all routes (a component with `code` and `message`).

Reply:
gotcha :+1: